lib/octokit/client/organizations.rb

```diff
@@ -595,6 +595,89 @@ def remove_organization_membership(org, options = {})
         user = options.delete(:user)
         user && boolean_from_response(:delete, "orgs/#{org}/memberships/#{user}", options)
       end
+
+      # Initiates the generation of a migration archive.
+      #
+      # Requires authenticated organization owner.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @option options [Array<String>] :repositories Repositories for the organization.
```

Comment:
Do you think `repositories` should be a full list of NWO or just the names of repositories? 

I could do `repositories.map { |short_repo_name| "#{org}/#{short_repo_name}" }` if the latter. 

/cc @pengwynn for design consideration.